guides/common/modules/ref_dhcp-isc-settings.adoc

```diff
@@ -0,0 +1,43 @@
+[id="DHCP_ISC_Settings_{context}"]
+= dhcp_isc Settings
+
+The _dhcp_isc_ provider uses a combination of the ISC DHCP server OMAPI management interface and parsing of configuration and lease files.
+This requires it to be run either on the same host as the DHCP server or to have network filesystem access to these files.
+You can define the following settings in `dhcp_isc.yml`:
+
+ifdef::foreman-el,katello[]
+.Configuring the path to the _config_ and _leases_ files on Enterprise Linux
```

Comment:
```suggestion
.Configuring the path to the _config_ and _leases_ files on {EL}
```

Or just keep it generic (and apply the same below to Debian):
```suggestion
.Configuring the path to the _config_ and _leases_ files
```

Reply:
Made it more generic.